### ISSUE TITLE
Fix Divider rendering in table

### ISSUE THREAD

**Issue by raineorshine** [cybersemics/em#543]
# Case 1

Normal view

## Steps to Reproduce

```
- a
  - =children
    - =pin
      - true
  - b
    - b1
    - ---
    - b2
  - c
    - This is a longer thought
    - ---
    - c2
```

## Current Behavior

Divider has a fixed width.

<img width="338" alt="image" src="https://github.com/cybersemics/em/assets/750276/3d9c9c30-e415-4f7b-a457-afd181fe6ee2">

## Expected Behavior

Divider width should be equal to the width of the widest sibling thought.

<img width="312" alt="image" src="https://github.com/cybersemics/em/assets/750276/1a52d07a-701c-4b5a-8b79-11257751e293">

# Case 2

Table View, Column 2, only child divider

## Steps to Reproduce

```
- Roadmap
  - =view
    - Table
  - Meta
    - Keyword Search
  - Adult Development
    - ---
  - Hypertext
    - Future of Text
    - ---
    - Rheingold
    - Landow
    - HT Best Of
    - HT '19
    - HT '18
    - HT '17
  - Semiotics
    - ---
```

## Current Behavior

<img width="389" alt="Brave Browser 2024-07-03 12 35 45" src="https://github.com/cybersemics/em/assets/750276/45392672-9853-42f4-a48e-a2c6b1f0d004">

## Expected Behavior

A divider *that is an only child* in column 2 of a table should be left aligned with the bullets and as wide as the widest col2 thought in the same table. 

In this example, `Adult Development` and `Semiotics` have an only-child divider, so their divider widths should be set to the widest width of all col2 thoughts in the table.

The divider in `Hypertext` is not an only-child, so it should be set to the width of the widest sibling, as usual.

<img width="393" alt="image" src="https://github.com/cybersemics/em/assets/750276/d8cb8e56-9954-49d7-bfe5-aa4eabd07870">

**Comment by incora-dev** [comment]
Estimate: 1.5h

**Comment by incora-dev** [comment]
@raineorshine For now, width for divider calculates like max-width of each subling element. But the problem occurs when we have a group with only divider(in this case we set 85px). What should we do it the case where we have only one divider in the group?

**Comment by raineorshine** [comment]
Ah, that makes sense. If it is the only-child in column 2 of a table, a divider should be the max-width of all of the column 2 thoughts.

Let's hold off on this for now though, it's just too low priority to spend time on.

**Comment by raineorshine** [comment]
- [ ] Blocked by #1842
